src/components/Nav/Nav.js

```diff
@@ -1,9 +1,97 @@
+import React from 'react';
 import styled from 'styled-components';
+import variables from '../../styles/variables';
+import MenuBar from '../../assets/Nav/MenuBar.png';

```

Comment:
```suggestion
import menuBar from '../../assets/Nav/MenuBar.png';
```
파일이름의 convention도 하나로 통일 해 주세요~ 이 파일만 PascalCase네요

Reply:
아... 무슨 생각으로 저렇게 했는지... 감사합니다!!

Reply:
don't 자책! 괜찮아요 그럴 수 있어요! 오타 낼 수도 있죠 뭐!